app/main/db/queries.py

```diff
@@ -54,69 +59,130 @@ def fuzzy_search(query_string):
         .distinct()
+        .order_by(Body.Name, Series.Name)
     )
-
+    query_results = None
     try:
-        result = db.session.execute(query)
+        query_results = db.session.execute(query)
+    except exc.SQLAlchemyError as e:
+        print("Failed to return results from database with error : " + str(e))
 
-        for r in result:
+    if query_results is not None:
+        for r in query_results:
             record = {
-                "TransferringBody": r.TransferringBody,
-                "Series": r.Series,
-                "ConsignmentReference": r.ConsignmentReference,
-                "FileName": r.FileName,
+                "transferring_body_id": r.BodyId,
+                "transferring_body": r.TransferringBody,
+                "series_id": r.SeriesId,
+                "series": r.Series,
+                "consignment_reference": r.ConsignmentReference,
+                "file_name": r.FileName,
             }
             results.append(record)
+    return results
+
+
+def browse_data(transferring_body_id=None, series_id=None, consignment_id=None):
+    results = []
+    if transferring_body_id is not None:
+        query = generate_transferring_body_filter_query(transferring_body_id)
+    elif series_id is not None:
+        query = generate_series_filter_query(series_id)
+    elif consignment_id is not None:
+        query = generate_consignment_reference_filter_query(consignment_id)
+    else:
+        query = generate_browse_everything_query()
+
+    query_results = None
+    try:
+        query_results = db.session.execute(query)
     except exc.SQLAlchemyError as e:
         print("Failed to return results from database with error : " + str(e))
+
+    if query_results is not None:
+        results = process_results(
+            query_results, transferring_body_id, series_id, consignment_id
+        )
     return results
 
 
-def get_file_data_grouped_by_transferring_body_and_series():
+def get_user_accessible_transferring_bodies(access_token):
+    unique_transferring_bodies = []
+    bodies = None
+    try:
+        bodies = Body.query.all()
+    except exc.SQLAlchemyError as e:
+        print("Failed to return results from database with error : " + str(e))
+
+    # get transferring bodies for which user has access to
+    user_transferring_body_groups = get_user_transferring_body_groups(
+        access_token
+    )
+
+    if bodies is not None and user_transferring_body_groups is not None:
+        for body in bodies:
+            group_name = body.Name
+            if len(user_transferring_body_groups) > 0:
+                for user_group in user_transferring_body_groups:
+                    if (
+                        user_group.strip().replace(" ", "").lower()
+                        == group_name.strip().replace(" ", "").lower()
+                    ):
+                        unique_transferring_bodies.append(group_name)
+    return unique_transferring_bodies
+
+
+def get_file_meta_data(file_id):
```

Comment:
```suggestion
def get_file_metadata(file_id):
```

Reply:
function name changed to get_file_metadata as requested